test/inductor/test_torchinductor_codegen_dynamic_shapes.py

```diff
@@ -136,6 +136,7 @@ def run(*ex, **kwargs):
     "test_zeros_dynamic_shapes": TestFailure(("cpu",)),
     "test_uint_dynamic_shapes": TestFailure(("cpu",)),
     "test_issue102546_dynamic_shapes": TestFailure(("cpu",)),
+    "test_adaptive_avg_pool1d_argmax_dynamic_shapes": TestFailure(("cpu",)),
```

Comment:
why does it fail for dynamic shapes?

Reply:
That's a good question. :-) It is a side effect from how the input size is picked and the usage of average pooling with a constant output size. The input tensor takes the size of [3,3,3] which would only have one symbol `s0` allocated for all these dims. Then according to the output size `2`, the input spatial dim can be deduced which makes `s0` concrete. I didn't realize it was that tricky. Thanks for the ask.

Reply:
Then, increasing a little bit the size of the input should fix this one I guess?

Reply:
Revised in PR: https://github.com/pytorch/pytorch/pull/113339